blog/_posts/2025-07-29-temporal-on-aks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Streamlining Temporal Worker Deployments on Azure Kubernetes Service (AKS)"
```

Comment:
Since this is the AKS blog, probably don't need to spell out "Azure Kubernetes Service" in the title.